src/TestPrerequisites/Common.Test.Prerequisites/Common.TestPrerequisites.csproj

```diff
@@ -0,0 +1,70 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage</TargetsForTfmSpecificBuildOutput>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);CopyPackageReferenceContent</TargetsForTfmSpecificContentInPackage>
+    <AssemblyName>Microsoft.Azure.PowerShell.TestPrerequisites</AssemblyName>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <RootNamespace>Microsoft.Azure.PowerShell.Test</RootNamespace>
+    <OutputPath>..\..\local-artifacts</OutputPath>
+    <IsPackable>true</IsPackable>
+    <HideReferenceDependencies>true</HideReferenceDependencies>
+    <Description>Wrapper test package to enable NetStandard testing</Description>
+    <AssemblyVersion>1.1.0.0</AssemblyVersion>
+    <FileVersion>1.1.0.0</FileVersion>
+  </PropertyGroup>
+
+ <Import Project="$(ProjectDir)..\..\..\tools\Common.Netcore.Dependencies.Test.targets" />
+
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Batch" Version="8.1.1" PrivateAssets="All" />
```

Comment:
Note: this is a stopgap for net472 testing: We should either generate this file from inspecting the csrpoj files for tests, or take all test dependencies from a single place.

Reply:
Why is this a `csproj` file? It looks like it could be done using a `targets` file.

Reply:
discussed offline